BUILD

```diff
@@ -0,0 +1,126 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "grpc_java_plugin",
+    srcs = [
+        "compiler/src/java_plugin/cpp/java_generator.cpp",
+        "compiler/src/java_plugin/cpp/java_generator.h",
+        "compiler/src/java_plugin/cpp/java_plugin.cpp",
+    ],
+    deps = [
+        "@com_google_protobuf//:protoc_lib",
+    ],
+)
+
+java_library(
+    name = "core",
+    srcs = glob([
+        "core/src/main/java/**/*.java",
```

Comment:
Please don't use super globs.  We are trying to split up the core and core internal package, and this pulls it in.   It also unnnessarily pulls in `inprocess` and `util`.